bee run error (get the go version  获取版本号出错)

bee run  error  获取版本号出错：

```
>bee run
go version go1.16.2 windows/amd64
go version go1.16.2 windows/amd64
panic: runtime error: index out of range [2] with length 1

goroutine 1 [running]:
github.com/beego/bee/v2/cmd/commands/version.GetGoVersion(0xc000091b40, 0xc0000d9170)
       **** /pkg/mod/github.com/beego/bee/v2@v2.0.2/cmd/commands/version/version.go:177 +0x1a6
```
reason:get not get go version
原因: 获取不了版本号


文件地址：
file path 
github.com\beego\bee\v2@v2.0.2\cmd\commands\version\version.go  line:168


```golang
func GetGoVersion() string {
	var (
		cmdOut []byte
		err    error
	)
	
	if cmdOut, err = exec.Command("go", "version").Output(); err != nil {
		beeLogger.Log.Fatalf("There was an error running 'go version' command: %s", err)
	}
	return strings.Split(string(cmdOut), " ")[2]
}
```
I can not ask how the porblom take place,but I  suggest replace by the code belows:
建议用下面代码替换


```golang
func GetGoVersion() string {
      return runtime.Version()
}
```

--- a/cmd/commands/version/version.go
+++ b/cmd/commands/version/version.go
@@ -6,15 +6,12 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
+	"gopkg.in/yaml.v2"
 	"io"
 	"os"
-	"os/exec"
 	path "path/filepath"
 	"regexp"
 	"runtime"
-	"strings"
-
-	"gopkg.in/yaml.v2"
 
 	"github.com/beego/bee/v2/cmd/commands"
 	"github.com/beego/bee/v2/config"
@@ -166,13 +163,5 @@ func GetBeegoVersion() string {
 }
 
 func GetGoVersion() string {
-	var (
-		cmdOut []byte
-		err    error
-	)
-
-	if cmdOut, err = exec.Command("go", "version").Output(); err != nil {
-		beeLogger.Log.Fatalf("There was an error running 'go version' command: %s", err)
-	}
-	return strings.Split(string(cmdOut), " ")[2]
+	return runtime.Version()
 }

